app/src/routes/api/v1/gfw.router.ts

```diff
@@ -0,0 +1,248 @@
+import { Context, Next } from 'koa';
+import Router from 'koa-router';
+import {
+    RWAPIMicroservice,
+    RequestToMicroserviceOptions,
+} from 'rw-api-microservice-node';
+import type request from 'request';
+
+import FieldSerializer from 'serializers/field.serializer';
+import logger from 'logger';
+import DatasetMiddleware from 'middleware/dataset.middleware';
+import GfwService from 'services/gfw.service';
+import ErrorSerializer from 'serializers/errorSerializer';
+
+const API_VERSION: string = 'v1';
```

Comment:
you are using this constant with multiple endpoints, meaning it will do more harm than good. I suggest removing it, and just using '/v1' in the different strings where it' currently being used.